inst/css/BS4/pkgdown.scss

```diff
@@ -86,7 +85,10 @@ a.anchor {
   }
 }
 
-
+/* Never uppercase the package name */
+.navbar-brand {
+  text-transform: none !important;
```

Comment:
At least one Bootswatch theme make that uppercase which is not good for a package name.